src/lib/BUTTON/devButton.cpp

```diff
@@ -29,9 +30,10 @@ static constexpr struct {
     bool pressType;
     uint8_t count;
     action_e action;
-} button_actions[2] = {
-    {true, 3, ACTION_START_WIFI},
-    {true, 7, ACTION_RESET_REBOOT}
+} button_actions[] = {
+    {true, 1, ACTION_BIND},
+    {true, 7, ACTION_START_WIFI},
+    {true, 14, ACTION_RESET_REBOOT}
```

Comment:
Don't forget that these are in half second increments + 1 extra half second for zero.
So your times are 1s for binding, 4 seconds for wifi and 7.5 seconds for reboot.

Reply:
You are 100% correct, no wonder they seemed so close together! I had forgotten.